### PR TITLE
KUI-1545: Updated Node version from 18 to 20

### PR DESCRIPTION
**Pull request by axelbjo**
Tested this by running locally and by using docker-compose up.

### CODE REVIEW

File: data-1724418229943.csv
Comment (karlandindrakryggen) [inline data-1724418229943.csv:1]: This file wasn't supposed to be included?

File: docker-compose-local-mongo.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
   mongodb:
     container_name: kursinfo-api-mongo
     image: mongo:4.0.0
     ports:
       - 27018:27017
     environment:
-      - MONGO_INITDB_ROOT_USERNAME=<CHOOSE_USERNAME>
-      - MONGO_INITDB_ROOT_PASSWORD=<CHOOSE_PASSWORD>
+      - MONGO_INITDB_ROOT_USERNAME=axel
+      - MONGO_INITDB_ROOT_PASSWORD=axel
```
Comment (karlandindrakryggen) [inline docker-compose-local-mongo.yml:9]: And this

**Review by karlandindrakryggen — CHANGES_REQUESTED** [review]

File: data-1724418229943.csv
Comment (axelbjo) [reply data-1724418229943.csv:1]: Same sneaky csv file as always.. Strange considering that I ran "git add Dockerfile package-lock.json package.json", but it must be from an older commit that I missed.

Comment (karlandindrakryggen) [reply data-1724418229943.csv:1]: it's easy to accidentally add extra files. I've try to have a habit of looking through my own PR directly when I have created it to find does misstakes. Like doing a fast PR review of my own code.

**Review by karlandindrakryggen — APPROVED** [review]

**Pull request merged**